<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sqlmap</string>
  </property>
  <widget class="QPlainTextEdit" name="text1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入URL</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入数据库名</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入表名</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入列名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textx1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>521</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../img/pic.qrc">:/新前缀/1.png</pixmap>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入等级</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入线程</string>
   </property>
  </widget>
  <zorder>text1</zorder>
  <zorder>text2</zorder>
  <zorder>text3</zorder>
  <zorder>text4</zorder>
  <zorder>comboBox</zorder>
  <zorder>button1</zorder>
  <zorder>textx1</zorder>
  <zorder>text5</zorder>
  <zorder>text6</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="../img/pic.qrc"/>
 </resources>
 <connections/>
</ui>
